### DISCUSSION TITLE
How to set rsync bwlimit in new version

### DISCUSSION THREAD

**Discussion by DrShotsPHD** [ericaltendorf/plotman#795] [Q&A] [answered]
In the old config there was a line for rsyncd_bwlimit: but the new version doesn't have it, I tried adding the line but the jobs are getting set to 80000 no matter what.

This is what my archiving section looks like now except I have removed my IP and user name.  Any idea where to set the bandwidth limit like in the previous version?

archiving:
        rsyncd_module: tjn_chia_node
        rsyncd_path: /mnt/node_usb
        rsyncd_host: 1.2.3.4
        rsyncd_user: userid

**Comment by racergoodwin** [comment, marked as answer]
At the moment I don't think there is a user configuration option in the config.yaml for that. But you can change it in line 34 of target_definitions.yaml.
I'm assuming linux based on the archiving snippet you posted above and you have followed the default installation instructions for chia and plotman, you should find it in ~/chia-blockchain/venv/lib/python3.*/site-packages/plotman/resources/
Substitute python3.* for the appropriate version you are using i.e. python3.8

**Reply by DrShotsPHD** [reply]
@racergoodwin, that did it!  I was looking through the source code last nighty and ran across the target_definitions.yaml file but didn't know where it was stored lol.
